Tests/UInt128Tests/UInt128Tests.swift

```diff
@@ -365,57 +372,34 @@ class UInt128UnsignedIntegerTests: XCTestCase {
             previousValue = currentValue
         }
     }
-    func testIndexTypes() {
```

Comment:
I'm not ready to lose this functionality yet. Before removing UInt128.successor() and UInt128.predecessor(), I would want to perform these changes in a new major release. I would like to defer this until the other changes are introduced.